spec/models/manageiq/providers/microsoft/infra_manager/refresher_spec.rb

```diff
@@ -52,10 +52,12 @@
   it "Classic same as graph refresh" do
     stub_settings_merge(:ems_refresh => {:scvmm => {:inventory_object_refresh => false}})
     EmsRefresh.refresh(@ems)
+    MiqQueue.destroy_all
```

Comment:
Needed to clear these because the second refresh just queued up "duplicates" of the same queue items, TODO should remove from the core test